### PR TITLE
Visualize 10 samples default.vds

### PR DESCRIPTION
**Pull request by yngve793**
Added python script for visualise the 10_samples_default data.

Suggestions for additional details to be added or adjustments will be appreciated. 
Notice that that int the 3D variant, the data can be rotated in 3 axis and boolean flags can be set to select what features to add to the plot.

### CODE REVIEW

File: testdata/10_samples/generate_plot3D.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    vertex_size = 15
+
+    plot_edges = True
+    edge_color = "green"
+
+    plot_data_values = False
+
+    plot_UTM_coordinates = True
+
+    fig = plt.figure(figsize=(18, 12))
```
Comment (achaikou) [inline testdata/10_samples/generate_plot3D.py:62]: Name the window aka figure? :innocent:

**Review by achaikou — COMMENTED** [review]
:heart: 
You are my favorite person of the day!

File: testdata/10_samples/generate_plot.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    ax.text(v[0]+0.0, v[1]+0.25, v[2], size="large")
+
+x_tics = np.arange(-1, 15, 1)
+y_tics = np.arange(-1, 12, 1)
+plt.xticks(x_tics)
+plt.yticks(y_tics)
+plt.xlim([-0.5, 14.5])
+plt.ylim([-0.5, 11.8])
+ax.grid(True)
+
```
[the remark this answers is not in the document]
Comment (yngve793) [reply testdata/10_samples/generate_plot.py:50]: Pep8 is happy. Added some spaces.

File: testdata/10_samples/generate_plot3D.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+    title_text = f"Visualization of: 10_samples_default"
+
+    plot_vertices = True
+    vertex_color = "blue"
+    vertex_size = 15
+
+    plot_edges = True
+    edge_color = "green"
+
```
[the remark this answers is not in the document]
Comment (yngve793) [reply testdata/10_samples/generate_plot3D.py:142]: Added a suggestion.

**Review by yngve793 — COMMENTED** [review]
Are we killing a mosquito with a sledgehammer here?

Update in accordance with comments.

**Review by achaikou — COMMENTED** [review]
What sledgehammer are you talking about? :smile: 
I only suggested implementing Vertex class for readability as it was difficult to follow which of 4 elements in v were responsible for what, so they had to be named, nothing else! :sweat_smile: 

But I am not going to complain about current code, it looks very nice! I also finally started to understand something in python plotting, so I feel all the efforts are justified! :slightly_smiling_face:

File: testdata/10_samples/generate_plot3D.py [hunk 4]
```diff
@@ -1,34 +1,93 @@
...
+    def __init__(self, xori, yori, xinc, yinc, rotation) -> None:
+        self.xori = xori
+        self.yori = yori
+        self.xinc = xinc
+        self.yinc = yinc
+        self.radian_rotation = rotation / 360 * 2 * np.pi
+
+    def index_to_utm(self, x, y):
+        complex_rotation = np.power(
```
[the remark this answers is not in the document]
Comment (yngve793) [reply testdata/10_samples/generate_plot3D.py:36]: Using Euler's formula for complex numbers

File: testdata/10_samples/generate_plot3D.py [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+
+    @property
+    def utm_coordinates(self):
+        return self.grid.index_to_utm(self.i, self.j)
+
+    def sample_at_depth(self, depth_value):
+        index = int((depth_value-self.samples.offset)/self.samples.step_size)
+        return self.data[index]
+
+    def coordinate_str(self, coordinate_type):
```
[the remark this answers is not in the document]
Comment (yngve793) [reply testdata/10_samples/generate_plot3D.py:73]: Depth is now an option.

**Review by yngve793 — COMMENTED** [review]
I definitely added weight to the hammer myself. I built on top of the vertex class idea and generalised. I think that it is better, but in the end it is just a drawing. 

Added dept as an option when printing label. 
Improved title to include label description.

**Review by achaikou — APPROVED** [review]
Awesome! Rebase, apply fixups and merge!

**Pull request merged**